ome-dundeeomero.yml

```diff
@@ -88,7 +88,7 @@
 
     #  Mock database user & creds, to allow Playbook to install
     #  OMERO, and allow for a manual PostgresSQL dump/restore.
-    - role: openmicroscopy.postgresql
+    - role: ome.postgresql
       postgresql_users_databases:
```

Comment:
There was a breaking change in the variable names: https://github.com/ome/ansible-role-postgresql/blob/master/CHANGES.md#changes-in-version-3
I think switching to the new variables is straightforward but make sure you've got a backup of the DB before just in case 😀

Reply:
Thanks, this had escaped my review. I will review the variable names and test the playbook in check-mode before getting this reviewed